### DISCUSSION TITLE
How to pass an empty list to the Hydra config when overriding?

### DISCUSSION THREAD

**Discussion by DrMatters** [facebookresearch/hydra#2447] [Q&A]
I'm trying to pass an empty list to the Hydra config but it keeps treating "[]" as an empty string. How can I pass an empty list using cli override?

**Comment by Jasha10** [comment]
For me, the following works:
```text
python my_app.py foo=[]
```
Perhaps your shell is misinterpreting the brackets?
You could try quoting the argument:
```text
python my_app.py 'foo=[]'
```
*Edit:* fixed quotation `'my_app.py foo=[]'` -> `my_app.py 'foo=[]'` (thanks Omry!)

**Reply by omry** [reply]
`python my_app.py 'foo=[]'`
